README.md

```diff
@@ -13,7 +13,14 @@ This environment includes the latest versions of the following software:
 
  - Docker (https://www.docker.com/products/docker-desktop)
  - Tested on macOS 10.15 (Catalina) and Windows 10 Version 10.0.18362
- - Windows 10 requires Docker File Sharing with the C drive to be enabled in the Docker settings, restart may be required after enabling sharing
+
+### Windows 10 Prerequisites
+
+- In Docker settings, file sharing must be enabled for the **C:\\** folder
+	- Located under the **Resources** section, then the **File Sharing** section as seen in [this image](https://i.imgur.com/maub2Wh.png).
+- If you do not see this option, make sure you are using **Windows containers**
+	- Right-click the Docker icon in the system tray and click **Switch to Windows containers** as seen in [this image](https://i.imgur.com/enYmIPH.png)
```

Comment:
Hey @asheroto, thanks for the contribution! More detailed instructions is always a good thing. One question, I'm running Linux containers just fine on Windows 10, were you not able to enable File Sharing without switching to Windows containers? Which version of Docker are you on?